src/lib/tabs/tab-group.spec.ts

```diff
@@ -647,15 +680,15 @@ describe('nested MatTabGroup with enabled animations', () => {
         (animationDone)="animationDone()"
         (focusChange)="handleFocus($event)"
         (selectedTabChange)="handleSelection($event)">
-      <mat-tab>
+      <mat-tab unique-name="tb01">
         <ng-template mat-tab-label>Tab One</ng-template>
         Tab one content
       </mat-tab>
-      <mat-tab>
+      <mat-tab unique-name="tb02">
         <ng-template mat-tab-label>Tab Two</ng-template>
         <span>Tab </span><span>two</span><span>content</span>
       </mat-tab>
-      <mat-tab>
+      <mat-tab  unique-name="tb03">
```

Comment:
Extra whitespace here.

```suggestion
      <mat-tab unique-name="tb03">
```

Reply:
done :)